conf.py

```diff
@@ -150,7 +150,12 @@ def codif_correction_cmd_linux(x):
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = "logo.png"
+project_path = pathlib.Path(__file__).parent.parent.resolve()
+project_path_string = project_path.name
+if project_path_string.__contains__("RTD-UM"):
```

Comment:
```suggestion
if project_path_string.__contains__("RTD-UM") or project_path_string.__contains__("RTD_UM"):
```
Existen ciertos manuales cuyo nombre es RTD_UM en lugar de RTD-UM. Lo más correcto sería corregir estos manuales para que su codificación sea la correcta (no nos tocaría a nosotros corregirlos sino a los responsables de dichos manuales), pero, mientras esta corrección sucede se puede controlar este problema añadiendo este caso en la condición del if